qiskit/circuit/quantumcircuit.py

```diff
@@ -1130,173 +1130,223 @@ def draw(self, output=None, scale=None, filename=None, style=None,
+                types in the output visualization. These items allow the use of
+                LaTeX formatting for gate names. The 'displaytext' dict can contain
+                any number of elements from one to the entire dict above.The default
+                values are::
 
                     {
-                        'id': 'id',
-                        'u0': 'U_0',
-                        'u1': 'U_1',
-                        'u2': 'U_2',
-                        'u3': 'U_3',
+                        'u1': '$\\mathrm{U}_1$',
+                        'u2': '$\\mathrm{U}_2$',
+                        'u3': '$\\mathrm{U}_3$',
+                        'u': 'U',
+                        'p': 'P',
+                        'id': 'I',
                         'x': 'X',
                         'y': 'Y',
                         'z': 'Z',
                         'h': 'H',
                         's': 'S',
-                        'sdg': 'S^\\dagger',
+                        'sdg': '$\\mathrm{S}^\\dagger$',
+                        'sx': '$\\sqrt{\\mathrm{X}}$',
+                        'sxdg': '$\\sqrt{\\mathrm{X}}^\\dagger$',
                         't': 'T',
-                        'tdg': 'T^\\dagger',
-                        'rx': 'R_x',
-                        'ry': 'R_y',
-                        'rz': 'R_z',
-                        'reset': '\\left|0\\right\\rangle'
+                        'tdg': '$\\mathrm{T}^\\dagger$',
+                        'dcx': 'Dcx',
+                        'iswap': 'Iswap',
+                        'ms': 'MS',
+                        'r': 'R',
+                        'rx': '$\\mathrm{R}_\\mathrm{X}$',
+                        'ry': '$\\mathrm{R}_\\mathrm{Y}$',
+                        'rz': '$\\mathrm{R}_\\mathrm{Z}$',
+                        'rxx': '$\\mathrm{R}_{\\mathrm{XX}}$',
+                        'ryy': '$\\mathrm{R}_{\\mathrm{YY}}$',
+                        'rzx': '$\\mathrm{R}_{\\mathrm{ZX}}$',
+                        'rzz': '$\\mathrm{R}_{\\mathrm{ZZ}}$',
+                        'reset': '$\\left|0\\right\\rangle$',
+                        'initialize': '$|\\psi\\rangle$'
                     }
 
-                You must specify all the necessary values if using this. There
-                is no provision for passing an incomplete dict in.
-            displaycolor (dict): The color codes to use for each circuit
-                element in the form (gate_color, text_color).
-                The default values are::
+            displaycolor (dict): the color codes to use for each circuit element in
+                the form (gate_color, text_color). Colors can also be entered without
+                the text color, such as 'u1': '#FA74A6', in which case the text color
+                will always be `gatetextcolor`. The `displaycolor` dict can contain
+                any number of elements from one to the entire dict above. The default
+                values are::
```

Comment:
Maybe avoid dumping the json file in the docstring?  We can refer to the file, know that there is one.

```suggestion
                any number of elements from one to the entire dict above. The default
                values are:: `default.json`
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/5223/commits/bfb8be30672ba6d81187f57c58a4079026e0bb0e